products/ansible.md

```diff
@@ -47,6 +47,16 @@ auto:
 # ansible-Core versions can be found on https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-changelogs
 # For Python / PowerShell versions, see https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#support-life
 releases:
+  - releaseCycle: "13"
+    ansibleCoreVersion: "2.20"
+    pythonVersionsControlNode: "3.12 - 3.14"
+    pythonVersionsManagedNode: "3.9 - 3.14"
+    powershellVersionsManagedNode: "5.1"
+    releaseDate: 2025-11-19
+    eol: false
+    latest: "13.2.0"
```

Comment:
```suggestion
    latest: "13.0.0"
```

13.0 seems to be the latest so far: https://github.com/ansible-community/ansible-build-data/blob/main/13/changelog.yaml